src/certificates_observer.py

```diff
@@ -35,9 +35,7 @@ def __init__(self, charm: CharmBaseWithState):
         self.framework.observe(
             self._charm.on.certificates_relation_joined, self._on_certificates_relation_joined
         )
-        self.framework.observe(
-            self.certificates.on.certificate_available, self._on_certificate_available
-        )
+        self.framework.observe(self.certificates.on.certificate_available, self._charm.reconcile)
```

Comment:
I think the guidance is that we need to match the method with the event name (like what you did before). Is there a reason for this change?

Reply:
We agreed in The Hague that this practice wasn't helpful, so this is now accepted